canvas_modules/common-canvas/src/common-canvas/canvas-controller.js

```diff
@@ -19,7 +19,7 @@ import AttachNodeToLinksAction from "../command-actions/attachNodeToLinksAction.
 import CommandStack from "../command-stack/command-stack.js";
 import ConvertSuperNodeExternalToLocal from "../command-actions/convertSuperNodeExternalToLocalAction.js";
 import ConvertSuperNodeLocalToExternal from "../command-actions/convertSuperNodeLocalToExternalAction.js";
-import constants from "./constants/canvas-constants";
+import * as constants from "./constants/canvas-constants";
```

Comment:
Pretty sure this was the root cause but the `_defineConstant` method was hiding it from the build.  That's why I changed all to using standard export instead.  After that the build failed until I fixed this.